libs/base/src/lib/components/forms/AssetExchange.tsx

```diff
@@ -99,7 +110,13 @@ export const AssetExchange: FC<Props> = ({
         decimals={inputDecimals}
       />
       <div>
-        <Arrow />
+        {isSwapPage ? (
+          <ExChangeButtonContainer>
+            <Button onClick={switchTokens}>↓</Button>
```

Comment:
nit: what do you think of using `↓↑`

Reply:
Good idea, it should be more intuitive for the users